components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -562,6 +589,19 @@ subroutine crm_physics_tend(ztodt, state, tend, ptend, pbuf, cam_in, cam_out, &
    logical(c_bool)             :: crm_accel_uv
    integer                     :: igstep
    integer                     :: idx_vt_t, idx_vt_q  ! variance transport constituent indices
+
+   ! pointers for crm_rad data on pbuf
+   real(crm_rknd), pointer :: crm_rad_tmp_qrad(:,:,:,:) ! rad heating
```

Comment:
I'd probably rather see these named like the variables in pbuf that they are going to refer to. I.e., `crm_qrad_tmp` instead of `crm_rad_tmp_qrad`. Or maybe better yet, `crm_qrad` and drop the `_tmp` altogether.

Reply:
I wanted to clearly distinguish between the "temporary" pointers and the actual rad variables. I guess I can drop the "tmp" though.

Reply:
done.